pinaxcon/registrasion/management/commands/populate_inventory.py

```diff
@@ -89,7 +89,7 @@ def populate_inventory(self):
             inv.Category,
             ("name",),
             name="Afternoon Tutorial Ticket (Thurs 3rd)",
-            description="Each tutorial has its own ticket.",
+            description="Each of our <a href=\"/program/tutorials/\">four tutorials</a> has its own ticket",
```

Comment:
We have two tutorial categories, one each for the morning and afternoon, so that people cannot accidentally buy clashing tutorial tickets.